src/stream/src/executor/stream_reader.rs

```diff
@@ -48,26 +46,14 @@ pub(super) struct StreamReaderWithPause<const BIASED: bool, M> {
 }
 
 impl<const BIASED: bool, M: Send + 'static> StreamReaderWithPause<BIASED, M> {
-    /// Receive messages from the reader. Hang up on error.
-    #[try_stream(ok = M, error = StreamExecutorError)]
-    async fn data_stream(stream: BoxTryStream<M>) {
```

Comment:
This looks like a no-op for `FusedStream`s. So I removed it.

Reply:
I also don't know why we need to do so before.